scripts/generate_options_schema.js

```diff
@@ -72,8 +77,10 @@ addTypeDocOptions({
                     ).map;
                 data.enum =
                     map instanceof Map
-                        ? [...map.keys()]
```

Comment:
This won't quite do the right thing for all enums, specifically [CommentStyle](http://typedoc.org/api/enums/CommentStyle.html#JSDoc). I think grabbing the value would be a better solution.